lib/infrastructure/data_access_layer/repository/firestore_database.dart

```diff
@@ -186,6 +186,11 @@ class FirestoreDatabase {
     });
   }
 
+  Future<bool> getAssetIdInfo({required String assetID}) async {
```

Comment:
why do we need a function just to check whether it exists ~ why not just get the data